constructor/winexe.py

```diff
@@ -59,6 +59,38 @@ def extra_files_commands(paths, common_parent):
         lines.append(f"File {path}")
     return lines
 
+def insert_tempfiles_commands(paths: os.PathLike) -> List[str]:
+    """Helper function that copies paths into temporary install directory.
+
+    Args:
+        paths (os.PathLike): Paths to files that need to be copied
+
+    Returns:
+        List[str]: Commands to be inserted into nsi template
+    """
+    if paths:
+        # Setting OutPath to PlugnsDir so NSIS File command copies the path into the PluginsDir
+        lines = ['SetOutPath $PLUGINSDIR']
+        for path in sorted([Path(p) for p in paths]):
+            lines.append(f"File {path}")
+        return lines
+    else:
+        return ['']
```

Comment:
```suggestion
    if not paths:
        return []
    # Setting OutPath to PluginsDir so NSIS File command copies the path into the PluginsDir
    lines = ['SetOutPath $PLUGINSDIR']
    for path in sorted([Path(p) for p in paths]):
        lines.append(f"File {path}")
    return lines
```

I usually deal with the least frequent cases first, then move to the general. Matter of taste, no strong opinion, but in this case I think it reads better.

Another question about this function:

- Why do we return `['']` instead of `[]`? I find it a bit unexpected and might deserve a comment if we really need to leave it as such.

Reply:
Hey Jaime, I guess it actually doesn't make a difference!  They both produce the same thing at the end of the day.